### ISSUE TITLE
SSH with Dev Containers requires `Docker Contexts: Use`

### ISSUE THREAD

**Issue by SerratedSharp** [microsoft/vscode-docs#7898]
This document(https://code.visualstudio.com/docs/containers/ssh#_set-up-ssh-tunneling) says you need Dev Containers and Remote - SSH extensions, but later references a command palette command that's provided by neither of these: `Docker Contexts: Use`.  I assume one of the various docker extensions provides this? Or do I need the full Docker Desktop client?

**Comment by ntrogh** [comment]
@SerratedSharp Yes, this command is provided by the [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) extension. Updating the docs to clarify this dependency.
